src/dashboards/Compact/index.tsx

```diff
@@ -108,6 +109,10 @@ const EnturDashboard = ({ history }: Props): JSX.Element => {
                             key={index.toString()}
                             data-grid={getDataGrid(index, maxWidthCols)}
                         >
+                            <ResizeHandle
+                                size="32px"
```

Comment:
Det vanlege er å bruke tall for pixelverdiar. Er det ein grunn til at det ikkje går i dette tilfellet?
```
size={32}
```

Reply:
Det skyldes at jeg har skrevet _mye_ css i det siste, så det lå i fingrene 😬 Fiksa 👌